javascript/javascript-5-switch.md

```diff
@@ -1,4 +1,4 @@
-# JavaScript 5
+# JavaScript 5 - Switch
```

Comment:
Make it `Switch Statements`

Reply:
have you seen the suggested changes button, @Kundan28 ?
![image](https://user-images.githubusercontent.com/29131689/75806243-992a3300-5d83-11ea-9907-c28fb4e150be.png)